### ISSUE TITLE
python3 / undefined symbol: PyInt_AsLong

### ISSUE THREAD

**Issue by tsomeq** [ArchangelSDY/PyPuzzle#12]
Hi.

I have installed  pypuzzle with pip.
Installation has correctly finished. but I cannot run example code on python3 due to an error.

```
 Traceback (most recent call last):
   File "puzzletest.py", line 1, in <module>
     import pypuzzle
  ImportError: /usr/lib64/python3.4/site-packages/pypuzzle.cpython-34m.s : undefined symbol: PyInt_AsLong
```

The error does not occur with using python 2 .

Perhaps it is necessary to wrap some function .
```
#if PY_MAJOR_VERSION >= 3
#  define PyInt_AsLong(x) (PyLong_AsLong((x)))
// ....
#endif
```

**Comment by ArchangelSDY** [comment]
Yes it does not support Python 3 yet. I'm considering doing this in next a few days.

**Comment by ArchangelSDY** [comment]
I've added Python3 support in master branch. Can you give it a try? If everything is fine I'll upload to PyPI.

**Comment by tsomeq** [comment]
Sorry to be late.

I have tested latest source, it seems to be fine.

During  compilation,  I got 1 warning like bellow.
It might be an ignorable problem.

```
building 'pypuzzle' extension
gcc -pthread -Wno-unused-result .... -Wall .... \
   -c src/pypuzzle.c -o build/temp.linux-x86_64-3.4/src/pypuzzle.o
src/pypuzzle.c: In function ‘main’:
src/pypuzzle.c:574:5:
  warning: passing argument 1 of ‘Py_SetProgramName’ 
   from incompatible pointer type [enabled by default]
     Py_SetProgramName(argv[0]);
     ^
In file included from /usr/include/python3.4m/Python.h:114:0,
                 from src/pypuzzle.c:1:
/usr/include/python3.4m/pythonrun.h:25:18: 
     note: expected ‘wchar_t *’ but argument is of type ‘char *’
 PyAPI_FUNC(void) Py_SetProgramName(wchar_t *);
                  ^
```

**Comment by ArchangelSDY** [comment]
Thanks! I've fixed the compiler warnings and released v1.3 to PyPI.
